core/consensus/babe/types/authority.hpp

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef KAGOME_AUTHORITY_HPP
+#define KAGOME_AUTHORITY_HPP
+
+#include "consensus/babe/common.hpp"
+#include "primitives/common.hpp"
+
+namespace kagome::consensus {
+  using AuthorityWeight = uint64_t;
+
+  /**
+   * Authority, which participate in Babe production
+   */
+  struct Authority {
```

Comment:
There is a struct WeightedAuthority in scheduled_change.hpp. This structure duplicates it. Please resolve that